src/components/inputs/z-input/index.tsx

```diff
@@ -254,7 +256,7 @@ export class ZInput {
         value={this.label}
         disabled={this.disabled}
         aria-label={this.label}
-        id={`${this.htmlid}_label`}
+        id={this.ariaLabel || `${this.htmlid}_label`}
```

Comment:
Sorry, I don't get why you set the id of the component as the "ariaLabel"

Reply:
there is no need, effectively.